internal/podman/client.go

```diff
@@ -457,20 +464,33 @@ func (o *client) Version(ctx context.Context) (version *Version, err error) {
 	return v, err
 }
 
-func (o *client) Logs(ctx context.Context) ([]interface{}, error) {
-	output, err := o.runPodman(ctx, "ps", "--format", "json")
+func (o *client) Logs(ctx context.Context) (map[string]interface{}, []error) {
+	l := map[string]interface{}{}
+	errs := []error{}
+
+	output, err := o.runPodman(ctx, "ps", "--all", "--format", "json", "--filter", "name=mongo")
```

Comment:
@blva I added the filter because I think the intention here was to get the conf of MongoDB containers only, or we want the entire conf?

Reply:
hello! yes, makes sense, here I added `podman ps` from @fmenezes suggestion, but I may have missed the filter part. FYI@fmenezes just in case you want the whole output for some reason.